src/pybind/mgr/dashboard/services/orchestrator.py

```diff
@@ -21,10 +21,10 @@ def instance(cls):
         return cls._instance
 
     def _call(self, method, *args, **kwargs):
-        if not Settings.ORCHESTRATOR_BACKEND:
+        _backend = mgr.remote("orchestrator_cli", "_select_orchestrator")
```

Comment:
Does this work if ``orchestrator_cli`` module is disabled. My experience is that ``mgr.remote`` does not work if the module is disabled. I'm currently running into this issue when calling ``mgr.remote('telemetry', 'get_module_option', 'config', '')`` and the telemetry module was disabled. It works when the module is enabled.
```
2019-02-08 14:05:34.893 7f8eea1dc700 -1 no module 'telemetry'
2019-02-08 14:05:34.913 7f8eea1dc700  0 mgr[dashboard] [::ffff:127.0.0.1:47522] [GET] [500] [3.989s] [admin] [1.4K] /api/mgr/module/telemetry
2019-02-08 14:05:34.913 7f8eea1dc700  0 mgr[dashboard] [b'{"status": "500 Internal Server Error", "detail": "The server encountered an unexpected condition which prevented it from fulfilling the request.", "traceback": "Traceback (most recent call last):\\n  File \\"/usr/lib/python3.7/site-packages/cherrypy/_cprequest.py\\", line 670, in respond\\n    response.body = self.handler()\\n  File \\"/usr/lib/python3.7/site-packages/cherrypy/lib/encoding.py\\", line 221, in __call__\\n    self.body = self.oldhandler(*args, **kwargs)\\n  File \\"/usr/lib/python3.7/site-packages/cherrypy/_cptools.py\\", line 237, in wrap\\n    return self.newhandler(innerfunc, *args, **kwargs)\\n  File \\"/ceph/src/pybind/mgr/dashboard/services/exception.py\\", line 88, in dashboard_exception_handler\\n    return handler(*args, **kwargs)\\n  File \\"/usr/lib/python3.7/site-packages/cherrypy/_cpdispatch.py\\", line 60, in __call__\\n    return self.callable(*self.args, **self.kwargs)\\n  File \\"/ceph/src/pybind/mgr/dashboard/controllers/__init__.py\\", line 541, in inner\\n    ret = func(*args, **kwargs)\\n  File \\"/ceph/src/pybind/mgr/dashboard/controllers/__init__.py\\", line 734, in wrapper\\n    return func(*vpath, **params)\\n  File \\"/ceph/src/pybind/mgr/dashboard/controllers/mgr_modules.py\\", line 46, in get\\n    key=name, default=option[\'default_value\'])\\n  File \\"/ceph/src/pybind/mgr/mgr_module.py\\", line 1187, in remote\\n    args, kwargs)\\nImportError: Module not found\\n", "version": "10.2.1"}']
```

Reply:
To allow modules to access settings from other modules https://github.com/ceph/ceph/pull/25651 has been introduced.